examples/install/Dockerfile.debian

```diff
@@ -0,0 +1,42 @@
+# This Dockerfile creates an debian image with avocado framework installed
```

Comment:
Maybe use examples/docker/ instead of examples/install?

Or add these files to contrib?